dags/glam.py

```diff
@@ -183,54 +132,42 @@
     dag=dag,
 )
 
+
 # This task runs first and replaces the relevant partition, followed
 # by the next task below that appends to the same partition of the same table.
-clients_daily_histogram_aggregates = bigquery_etl_query(
-    task_id="clients_daily_histogram_aggregates",
-    destination_table="clients_daily_histogram_aggregates_v1",
-    dataset_id=dataset_id,
+clients_daily_histogram_aggregates_parent = generate_and_run_query(
+    task_id="clients_daily_histogram_aggregates_parent",
     project_id=project_id,
-    owner="msamuel@mozilla.com",
-    email=[
-        "telemetry-alerts@mozilla.com",
-        "msamuel@mozilla.com",
-        "robhudson@mozilla.com",
-        "bewu@mozilla.com",
-    ],
-    arguments=("--replace",),
-    parameters=(
-        "sample_size:INT64:{}".format(PERCENT_RELEASE_WINDOWS_SAMPLING),
-    ),
+    source_dataset_id=dataset_id,
+    sample_size=PERCENT_RELEASE_WINDOWS_SAMPLING,
+    overwrite=True,
+    probe_type="histogram",
+    process="parent",
+    get_logs=False,
     dag=dag,
 )
 
-clients_daily_keyed_histogram_aggregates = gke_command(
-    task_id="clients_daily_keyed_histogram_aggregates",
-    owner="msamuel@mozilla.com",
-    email=[
-        "telemetry-alerts@mozilla.com",
-        "msamuel@mozilla.com",
-        "robhudson@mozilla.com",
-        "bewu@mozilla.com",
-    ],
-    cmds=["bash"],
-    env_vars={
-        "PROJECT": project_id,
-        "PROD_DATASET": dataset_id,
-        "DATASET": dataset_id,
-        "SUBMISSION_DATE": "{{ ds }}",
-        "RUN_QUERY": "t",
-        "APPEND": "t",
-    },
-    command=[
-        "script/glam/generate_and_run_desktop_sql",
-        "keyed_histogram",
-        PERCENT_RELEASE_WINDOWS_SAMPLING,
-    ],
-    docker_image="mozilla/bigquery-etl:latest",
-    gcp_conn_id="google_cloud_derived_datasets",
+clients_daily_histogram_aggregates_content = generate_and_run_query(
+    task_id="clients_daily_histogram_aggregates_content",
+    project_id=project_id,
+    source_dataset_id=dataset_id,
+    sample_size=PERCENT_RELEASE_WINDOWS_SAMPLING,
+    overwrite=False,
+    probe_type="histogram",
+    process="content",
+    get_logs=False,
```

Comment:
Do we still need this now that the query lengths are smaller?

Reply:
It's possible to remove it but it's cutting really close.  If the query was about a minute longer it would fail so I would keep it

Reply:
Actually I just tested and both the content and keyed queries just failed after 7 minutes but the parent query succeeded after 8 minutes so definitely should keep get_logs=False